strapi/gcp-deployment/main.tf

```diff
@@ -0,0 +1,130 @@
+###############
+## Variables ##
+###############
+
+variable "project_name" {
+  type = string
+  default = "ch-test-strapi"
+}
+
+variable "location" {
+  type = string
+  default = "asia-southeast1"
+}
+
+variable "db_name" {
+  type = string
+  default = "strapi"
+}
+
+variable "db_instance_name" {
+  type = string
+  default = "strapi-db-instance"
+}
+
+variable "app_bucket_name" {
+  type = string
+  default = "ch-test-strapi-app-bucket"
+  # default = "ch-test-strapi.appspot.com"
+}
+
+variable "db_username" {
+  type = string
+  sensitive = true
+}
+
+variable "db_password" {
+  type = string
+  sensitive = true
+}
+
+######################
+## Project Settings ##
+######################
+
+data "google_billing_account" "account" {
+  display_name = "My Billing Account"
+  open = true
+}
+
+resource "google_project" "project" {
+  name = var.project_name
+  project_id = var.project_name
+  org_id = "257845927637" # better.sg
+  billing_account = data.google_billing_account.account.id
+}
+
+// Enable Cloud SQL Admin API to connect to database
+resource "google_project_service" "sql_admin_api" {
+  project = google_project.project.project_id
+  service = "sqladmin.googleapis.com"
+}
+
+#############
+## Storage ##
+#############
```

Comment:
These headers are great. `.tf` files have a tendency to get unreadable otherwise.